test/core/crypto/hasher/hasher_test.cpp

```diff
@@ -0,0 +1,143 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <gtest/gtest.h>
+#include <stdio.h>
+
+#include "common/blob.hpp"
+#include "crypto/hasher/hasher_impl.hpp"
+
+/**
+ * Hasher fixture
+ */
+class Hasher : public testing::Test {
+ public:
+  using Buffer = kagome::common::Buffer;
```

Comment:
You can just write `using kagome::common::Buffer` outside the class

Reply:
Ok.